app/scripts/views/mixins/signin-mixin.js

```diff
@@ -147,19 +148,29 @@ define(function (require, exports, module) {
 
     _engageSignInForm (event) {
       if (event && event.target) {
-        var target = this.$el.find(event.target).attr('id');
-
-        if (target === 'have-account') {
-          // if the user clicks on 'have-account' we count that as flow event instead of the 'engage' event.
-          // Details: https://github.com/mozilla/fxa-content-server/pull/4221
-          this.logEvent('flow.have-account');
-          return;
+        const target = this.$el.find(event.target).attr('id');
```

Comment:
Can you just do an `const target = event.target.id` or `const target = event.target.getAttribute('id')`? I dunno if either of those are easier to read.